ios/Video/Features/RCTVideoUtils.swift

```diff
@@ -299,23 +299,30 @@ enum RCTVideoUtils {
                     if let textTracks {
                         for i in 0 ..< tracks.count {
                             guard let track = tracks[i]?.first else { continue } // fix when there's no textTrackAsset
-                            validTextTracks.append(textTracks[i])
 
                             let textCompTrack: AVMutableCompositionTrack! = mixComposition.addMutableTrack(withMediaType: AVMediaType.text,
                                                                                                            preferredTrackID: kCMPersistentTrackID_Invalid)
-                            try? textCompTrack.insertTimeRange(
-                                CMTimeRangeMake(start: .zero, duration: videoAsset.timeRange.duration),
-                                of: track,
-                                at: .zero
-                            )
+
+                            do {
+                                try textCompTrack.insertTimeRange(
+                                    CMTimeRangeMake(start: .zero, duration: videoAsset.timeRange.duration),
+                                    of: track,
+                                    at: .zero
+                                )
+                                validTextTracks.append(textTracks[i])
+                            } catch {
+                                continue
+                            }
```

Comment:
one downside here, is we fail silently here. What do you recommend ?

Reply:
I think we can leave TODO and later when we will "upgrade errors" we will throw here 
Eventually we cloud warn user that something is failing